cypress-tests/cypress/support/commands.js

```diff
@@ -883,6 +968,16 @@ Cypress.Commands.add(
         ).to.be.null;
         expect(response.body.connector_mandate_id, "connector_mandate_id").to.be
           .null;
```

Comment:
3.status = 'requires_payment_method'

We are already checking this in each connector.js file
expect(res_data.body[key]).to.equal(
            response.body[key],
            `Expected ${res_data.body[key]} but got ${response.body[key]}`
);

6.check if the profile id is the expected one instead of doing null check
      Its a random created profile_id and even we are getting profile id from the global state so it will fail if the profile id is not present or its not stored in globalstate.
      
4.Check deep equals (nested object should be covered, eg: JSON.stringify(obj1) === JSON.stringify(obj2)) for the shipping address
5.Check deep equals for billing address
  - Billling and shipping address we are not passing in the intent create. We are passing only the mandatory field in payment intent create. So we are verifying these checks in confirm and other calls
  
 7.external_3ds_authentication_attempted is same as the request body
   As we have not added these scenario's yet so havent added these checks in the response once we have implemented that feature we will include these cases.

Reply:
3. rename the actual and expected response fields properly. res_data and response don't convey the exact meaning
6. We need tests for paymentIntentCreate with a specific profile ID because merchants can choose to make payments in a particular profile
4. PaymentIntentCreate is the only place a merchant can send a customer's existing billing address; we need to validate it here. We need to think of these scenarios
-    Payment intent without billing address and all required fields being null in /account/payment_methods call
-  Payment intent with billing address and all required fields are preset in /account/payment_methods call
-  Payment intent with partial billing address and respective fields are preset/null in  /account/payment_methods call
7. Add this validation for Netcetera as it is planned for this sprint itself, or add a TODO with the issue

Reply:
3,4,5 We will take this in the seperate PR.
7 We will plan in this sprint